OrbitCore/TracepointEventBuffer.cpp

```diff
@@ -22,6 +23,7 @@ void TracepointEventBuffer::AddTracepointEventAndMapToThreads(uint64_t time,
     event.set_pid(process_id);
     event.set_cpu(cpu);
     event_map_tracepoints_not_in_target_process[time] = std::move(event);
+    no_tracepoints_not_in_target_process_++;
```

Comment:
You don't really need this: 
`tracepoint_events_[kNotTargetProcessThreadId].size()`
will be the same number.

Reply:
Thank you so much for pointing that out!